tasks_management/signals.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+
+from core.service_signals import ServiceSignalBindType
+from core.signals import bind_service_signal
+from tasks_management.models import TaskGroup, Task
+
+
+def resolve_task_any(task, business_status: Dict[str, str]):
+    if any(status == "APPROVED" for status in business_status.values()):
+        return True
+    return False
+
+
+def resolve_task_all(task, business_status: Dict[str, str]):
+    approve_count = sum(1 for status in business_status.values() if status == "APPROVED")
+    group_members = TaskGroup.objects.get(task=task).taskexecutor_set.count()
+    if approve_count == group_members:
+        return True
+    return False
+
+
+def resolve_task_n(task, business_status: Dict[str, str]):
+    approve_count = sum(1 for status in business_status.values() if status == "APPROVED")
+    n = 1  # hardcoded for now
+    if approve_count >= n:
```

Comment:
Same as in resolve_task_all regarding if/return statements

Reply:
resolved